draft-ietf-quic-recovery.md

```diff
@@ -268,6 +268,13 @@ packets. While this increases network load, it is considered safe, since the
 sender will still reduce its sending rate exponentially under persistent
 congestion ({{pto}}).
 
+## Handshake packets are not special
+
+TCP reduces the congestion window to one packet when the SYN or SYN/ACK
+are lost ({{?RFC5682}}).  QUIC treats loss of a packet containing handshake
+data the same as other losses and reduces the congestion window.  When the
+PTO timer expires during the handshake, one or two packets are sent and
+the timer is doubled.
```

Comment:
Another suggestion for the first sentence:
TCP treats the loss of SYN or SYN-ACK packet as persistent congestion and reduces the congestion window to one packet.